app/javascript/packages/password-toggle/password-toggle.tsx

```diff
@@ -1,91 +0,0 @@
-import { forwardRef } from 'react';
-import type { HTMLAttributes, ForwardedRef } from 'react';
-import { t } from '@18f/identity-i18n';
-import { TextInput } from '@18f/identity-components';
```

Comment:
This appears to be the only instance of `TextInput`, so we can probably remove it as well.

Reply:
Removed in 80f248a08.